projects/fsastorefrontlib/src/core/my-account/store/reducers/quote.reducer.ts

```diff
@@ -3,24 +3,35 @@ import { QuoteState } from '../my-account-state';
 
 export const initialState: QuoteState = {
   quotes: [],
+  quoteDetails: {},
   loaded: false,
 };
 
 export function reducer(
   state = initialState,
   action: fromAction.QuoteAction
 ): QuoteState {
-  if (action.type === fromAction.LOAD_QUOTES_SUCCESS) {
-    const quotes = action.payload ? [...action.payload] : [];
-    return {
-      ...state,
-      quotes,
-      loaded: true,
-    };
+  switch (action.type) {
+    case fromAction.LOAD_QUOTES_SUCCESS: {
+      const quotes = action.payload ? [...action.payload] : [];
```

Comment:
Is there a need for ternary here? Can we just use `const quotes = [...action.payload];`?

Reply:
Actually I didn't write that part, just include a part regarding loading quote details, but it can be refactored for sure

Reply:
I know, but maybe we don't need it any more, whoever wrote it 🙂

Reply:
Sure, will refactor it 👍

Reply:
Done!